ganttproject/src/net/sourceforge/ganttproject/ResourceTreeTable.java

```diff
@@ -123,12 +131,93 @@ protected Chart getChart() {
     return myUiFacade.getResourceChart();
   }
 
+  private static class AscendingNameComparator implements Comparator<HumanResource> {
```

Comment:
Maybe use just one class? There is `reversed` method in any `Comparator` which makes, well, reversed comparator.